bundles/org.openhab.binding.somneo/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -14,6 +14,23 @@
 			<channel-group id="sunset" typeId="sunset"/>
 			<channel-group id="relax" typeId="relax"/>
 			<channel-group id="audio" typeId="audio"/>
+			<channel-group id="alarm" typeId="alarmCommon"/>
+			<channel-group id="alarm1" typeId="alarm"/>
+			<channel-group id="alarm2" typeId="alarm"/>
+			<channel-group id="alarm3" typeId="alarmErasable"/>
+			<channel-group id="alarm4" typeId="alarmErasable"/>
+			<channel-group id="alarm5" typeId="alarmErasable"/>
+			<channel-group id="alarm6" typeId="alarmErasable"/>
+			<channel-group id="alarm7" typeId="alarmErasable"/>
+			<channel-group id="alarm8" typeId="alarmErasable"/>
+			<channel-group id="alarm9" typeId="alarmErasable"/>
+			<channel-group id="alarm10" typeId="alarmErasable"/>
+			<channel-group id="alarm11" typeId="alarmErasable"/>
+			<channel-group id="alarm12" typeId="alarmErasable"/>
+			<channel-group id="alarm13" typeId="alarmErasable"/>
+			<channel-group id="alarm14" typeId="alarmErasable"/>
+			<channel-group id="alarm15" typeId="alarmErasable"/>
+			<channel-group id="alarm16" typeId="alarmErasable"/>
```

Comment:
Yes, the device API provides a static list of 16 alarms. The first two are always present and cannot be reset (they do not have the configured switch). The others can be reset to the default values.
For comparison the mobile app from Philips works like the standard alarm clock app. Up to 16 alarms can be added and all except the first two can be deleted. Unfortunately, I could not map this user experience in OpenHAB.

Reply:
Can you paste a screenshot how these channel groups like in the UI? Would save me some time trying to understand. 🙂

Reply:
![image](https://github.com/openhab/openhab-addons/assets/20473294/638d038b-afb7-4b2f-a197-c875ef94d55f)

Reply:
Thanks! I think it confirms my suspicion for posting the comment: Shouldn't there be individual labels for the channel groups so that they can be distinguished not only by channel UIDs? I.e. "Wecker 2", "Wecker 3" etc. in the group titles above.